### PR TITLE
Added support for MIFARE Ultralight EV1 password authentication

### PR DESCRIPTION
**Pull request by scriptcoffee**
Added wrapper for phalMful_PwdAuth() function to be used for password authentication on MIFARE Ultralight EV1 cards.

### CODE REVIEW

**Comment by svvitale** [comment]
This looks good, @scriptcoffee.  Would you mind adding some sample usage to the README?  I'll get it merged straight away.

**Comment by scriptcoffee** [comment]
I've added a section explaining how to set and unset password protection and how to authenticate. Addresses used are specific to the NTAG216 card as this is the card I used in my project. I hope it's understandable how to use it.

File: README.md [hunk 1]
```diff
@@ -96,6 +96,68 @@ ndef_data = mifare.read_ndef()
...
 
+**Authentication example:**<br />
+This example uses the address layout of a NTAG216 card. Please refer to the specific card manual for the address layout.<br />
+NTAG cards remain authenticated until removed from field or an error occurs. Reauthenticate to gain access again.
+
+NTAG216 configuration page layout:
+
+| Address  | Byte 0 | Byte 1 |    Byte 2    | Byte 3 |
+| -------- | ------- | ----- | ------------ | ------ |
+|   0xE3   | Mirror  | RFUI  | Morror_Page  | AUTH0  |
```
Comment (svvitale) [inline README.md]: Should byte 2 be "Mirror_Page"?

**Comment by svvitale** [comment]
Documentation makes perfect sense.  I pointed out one little typo.

**Comment by scriptcoffee** [comment]
Oh, I must have missed that one.

**Pull request merged**